pkg/operator/ceph/cluster/osd/provision_spec.go

```diff
@@ -320,3 +325,11 @@ func (c *Cluster) provisionOSDContainer(osdProps osdProperties, copyBinariesMoun
 
 	return osdProvisionContainer, nil
 }
+
+func SetAllowLoopDevices(value string) {
```

Comment:
Besides setting this on the pod specs, I don't think we should ever set this env var except during testing

Reply:
I believe this is resolved in the discussion at https://github.com/rook/rook/pull/11268#discussion_r1017495887 .